test/fuzz/transformation_swap_function_variables_test.cpp

```diff
@@ -185,24 +186,40 @@ TEST(TransformationSwapFunctionVariables, IsApplicable) {
 
   // Successful transformations
   {
+    auto first_instruction = context->get_def_use_mgr()->GetDef(24);
+    auto second_instruction = context->get_def_use_mgr()->GetDef(28);
     // Swap two OpVariable instructions in the same function.
     TransformationSwapFunctionVariables transformation(24, 28);
+
     ASSERT_TRUE(
         transformation.IsApplicable(context.get(), transformation_context));
+
     ApplyAndCheckFreshIds(transformation, context.get(),
                           &transformation_context);
+
     ASSERT_TRUE(fuzzerutil::IsValidAndWellFormed(
         context.get(), validator_options, kConsoleMessageConsumer));
+
+    ASSERT_EQ(first_instruction, context->get_def_use_mgr()->GetDef(28));
+    ASSERT_EQ(second_instruction, context->get_def_use_mgr()->GetDef(24));
```

Comment:
Once you have made the change above per my suggestion, I guess you will need to swap these back! :-) And the same below.

Reply:
Done.